awx/wsgi.py

```diff
@@ -41,41 +40,12 @@
             still works".format(social_django.__version__))
 
 
-if django.__version__ != '1.11.20':
-    raise RuntimeError("Django version other than 1.11.20 detected {}. \
+if not django.__version__.startswith('1.'):
+    raise RuntimeError("Django version other than 1.XX detected {}. \
             Inherit from WSGIHandler to support short-circuit Django Middleware. \
-            This is known to work for Django 1.11.20 and may not work with other, \
+            This is known to work for Django 1.XX and may not work with other, \
             even minor, versions.".format(django.__version__))
 
 
-if settings.MIDDLEWARE:
-    raise RuntimeError("MIDDLEWARE setting detected. \
-            The 'migration in progress' view feature short-circuits OLD Django \
-            MIDDLEWARE_CLASSES behavior. With the new Django MIDDLEWARE beahvior \
-            it's possible to short-ciruit the middleware onion through supported \
-            middleware mechanisms. Further, from django.core.wsgi.get_wsgi_application() \
-            should be called to get an instance of WSGIHandler().")
```

Comment:
I thought the removal of this assumed that the `MigrationRanCheckMiddleware` class short-circuited. In the current state, it looks like it doesn't.

Reply:
After looking at the internals of Django's newer middleware processing, I am reasonably sure that `MigrationRanCheckMiddleware` will short-circuit under the appropriate circumstance.